src/backgroundblurprocessor/BackgroundBlurProcessorProvided.ts

```diff
@@ -303,8 +333,8 @@ export default class BackgroundBlurProcessorProvided implements BackgroundBlurPr
 
       if (this.scaledCanvas === undefined) {
         this.scaledCanvas = document.createElement('canvas');
-        this.scaledCanvas.width = inputCanvas.width * hscale;
-        this.scaledCanvas.height = inputCanvas.height * vscale;
+        this.scaledCanvas.width = this.spec.model.input.width;
```

Comment:
Is this change related? If not, could we document that in CHANGELOG :)

Reply:
This is unrelated. I will update in changelog.